layers/meta-balena-raspberrypi/recipes-connectivity/bluez5/bluez5/Move-the-43xx-firmware-into-lib-firmware.patch

```diff
@@ -0,0 +1,21 @@
+From: Alex Gonzalez <alexg@balena.io>
+Date: Mon, 23 Jan 2023 17:18:43 +0100
+Subject: Move the 43xx firmware into /lib/firmware
+
+---
+ tools/hciattach.h | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/tools/hciattach.h b/tools/hciattach.h
+index dfa4c1e7abe7..e8848476609d 100644
+--- a/tools/hciattach.h
++++ b/tools/hciattach.h
+@@ -41,7 +41,7 @@
+ #define HCI_UART_VND_DETECT	5
+ 
+ #ifndef FIRMWARE_DIR
+-#define FIRMWARE_DIR "/etc/firmware"
++#define FIRMWARE_DIR "/lib/firmware"
```

Comment:
In meta-balena the bluez5 recipe now includes:
```
CFLAGS += "-DFIRMWARE_DIR=\\"${nonarch_base_libdir}/firmware\\""
```
So this is probably no longer required.